docs/markdown/whisker-series.md

```diff
@@ -0,0 +1,50 @@
+## WhiskerSeries
+
+<!-- INJECT:"WhiskerScatterplotChart" -->
+
+The WhiskerSeries is much like the MarkSeries, with the addition of variance "whiskers" on each mark.
+
+Deploy a WhiskerSeries like this:
+
+```javascript
+render() {
+  return (
+    <XYPlot
+      width={300}
+      height={300}>
+      <WhiskerSeries
+        className="whisker-series-example"
+        sizeRange={[5, 15]}
+        data={myData}/>
+    </XYPlot>
+  );
+```
+
+Just like other series, WhiskerSeries expects its data to be formatted as an array of
+objects. These data points may incude an `xVariance` property, a `yVariance` property,
+or both:
+
+```javascript
+const myData = [
+  {x: 1, y: 10, xVariance: 4, yVariance: 4},
+  {x: 1.7, y: 12, xVariance: 7, yVariance: 7},
+  {x: 2, y: 5, xVariance: 3, yVariance: 3},
+  {x: 3, y: 15, xVariance: 10, yVariance: 10},
+  {x: 2.5, y: 7, xVariance: 4, yVariance: 4}
+];
+```
+
+WhiskerSeries also accepts a `size` value that specifies an empty "buffer" region.
+This is especially useful if you are combinging the whiskers with another series, and
+the marks of that series include transparent regions. The buffer region prevents
+whisker lines from being drawn behind that other mark.
+
+```javascript
+const myData = [
+  {x: 1, y: 10, size: 30, yVariance: 4},
+  {x: 1.7, y: 12, size: 10, yVariance: 7},
+  {x: 2, y: 5, size: 1, yVariance: 3},
+  {x: 3, y: 15, size: 12, yVariance: 10},
+  {x: 2.5, y: 7, size: 4, yVariance: 4}
+];
+```
```

Comment:
To be completionist, you need to have the API reference in here. Feel free to copy from the MarkSeries reference and remove&add section as necessary